src/bitcoin-tx.cpp

```diff
@@ -206,7 +206,7 @@ static void MutateTxAddInput(CMutableTransaction& tx, const string& strInput)
     // extract the optional sequence number
     uint32_t nSequenceIn=std::numeric_limits<unsigned int>::max();
     if (vStrInputParts.size() > 2)
-        nSequenceIn = atoi(vStrInputParts[2]);
+        nSequenceIn = std::stoul(vStrInputParts[2]);
```

Comment:
IMO we only support platforms/archs where int is always 32bit and I think `std::numeric_limits<uint32_t>::max()` won't change much.